frontend/dockerfile/linter/generate.go

```diff
@@ -35,6 +36,12 @@ aliases:
   - {{ .Rule.URLAlias }}
 {{- end }}
 ---
+{{- if .Rule.Experimental }}
```

Comment:
nit: add empty line after front matter 
```suggestion
{{- if .Rule.Experimental }}

```

Reply:
👍 Fixed!